src/hotspot/share/prims/methodHandles.cpp

```diff
@@ -1257,54 +1257,9 @@ JVM_ENTRY(void, MHN_copyOutBootstrapArguments(JNIEnv* env, jobject igcls,
       != caller->constants()->bootstrap_argument_count_at(bss_index_in_pool)) {
       THROW_MSG(vmSymbols::java_lang_InternalError(), "bad index info (1)");
   }
-  objArrayHandle buf(THREAD, (objArrayOop) JNIHandles::resolve(buf_jh));
```

Comment:
Do you want to assert about the value of 'start'  and 'end'?  This became a nice cleanup.

Reply:
Yes, we do!